sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -549,3 +552,33 @@ def service_properties_deserialize(generated: Any) -> Dict[str, Any]:
         'minute_metrics': Metrics._from_generated(generated.minute_metrics),  # pylint: disable=protected-access
         'cors': [CorsRule._from_generated(cors) for cors in generated.cors],  # pylint: disable=protected-access
     }
+
+class StorageConfiguration(Configuration):
```

Comment:
This should go in `_shared.models`. Once you do that you can't include anything from the Queue package in here directly like `message_encode_policy`/`message_decode_policy` so we'll have to figure out something else for those.